cowprotocol/cowswap
# [Limit orders] confirm screen

# Summary

<img width="455" alt="image" src="https://user-images.githubusercontent.com/7122625/197396728-7b1bc506-4037-4987-8eb8-9c6b74504104.png">

## Test
1. Select valid trade (input/output currencies and amounts)
2. Ensure that `/api/v1/quote` didn't respose with 400 error (we will get rid of the call in the future)
3. Click `Trade`
